Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -1647,8 +1647,37 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(transformerTensor.shape, transformerResult.shape)
     }
 
+    func testConformArraytoLayer(){
+        var add_layer:[Dense<Float>] = []
```

Comment:
The name `add_layer` doesn't quite make sense. It sounds like the name of a function, but it's an array.
```suggestion
        var layers: [Dense<Float>] = []
```